### DISCUSSION TITLE
Why is the wildcard random instead of filling in the prompts in order?

### DISCUSSION THREAD

**Discussion by lost9999** [adieyal/sd-dynamic-prompts#636] [Q&A]
I hope the wildcard can fill in the prompts in order, please tell me how I should achieve this, thank you!

**Comment by akx** [comment]
You might want Combinatorial generation: https://github.com/adieyal/sd-dynamic-prompts#combinatorial-generation

**Reply by lost9999** [reply]
Combinatorial generation only generates all the prompts in the wildcard, but it is not generated in the order of the prompts listed in the wildcard.

**Reply by NeoAnthropocene** [reply]
It's been a while you have asked this question but your answer should be this:
https://github.com/adieyal/sd-dynamic-prompts/blob/main/docs/SYNTAX.md#cyclical-sampler
